stablehlo/conversions/tosa/transforms/StablehloLegalizeToTosa.cpp

```diff
@@ -523,6 +523,39 @@ struct ConvertStablehloReshapeOp
   }
 };
 
+struct ConvertStablehloFloatDivideOp
+    : public OpRewritePattern<stablehlo::DivOp> {
+  using OpRewritePattern<stablehlo::DivOp>::OpRewritePattern;
+
+  LogicalResult matchAndRewrite(stablehlo::DivOp op,
+                                PatternRewriter& rewriter) const override {
+    auto lhsType = dyn_cast<RankedTensorType>(op.getLhs().getType());
+    auto rhsType = dyn_cast<RankedTensorType>(op.getRhs().getType());
+    if (!lhsType || !rhsType) {
+      return rewriter.notifyMatchFailure(op, "expected ranked tensor types");
+    }
+
+    if (!llvm::isa<mlir::FloatType>(lhsType.getElementType()) &&
+        !llvm::isa<mlir::FloatType>(rhsType.getElementType())) {
+      return rewriter.notifyMatchFailure(
+          op, "only converts floating point division");
+    }
+
+    auto shiftTensorType = RankedTensorType::get({1}, rewriter.getI8Type());
+    auto zeroShiftValue = DenseElementsAttr::get(
+        shiftTensorType, rewriter.getIntegerAttr(rewriter.getI8Type(), 0));
+    auto shiftConst = rewriter.create<tosa::ConstOp>(
```

Comment:
```
rewriter.create<tosa::ReciprocalOp>(op.getLoc(), rhsType, op.getRhs());
==>
tosa::ReciprocalOp::create(rewriter, op.getLoc(), rhsType, op.getRhs());
```

For all calls to "create<...>"

We recently fixed this in the rest of the StableHLO codebase